compose/material3/material3/src/skikoMain/kotlin/androidx/compose/material3/WideNavigationRail.skiko.kt

```diff
@@ -42,12 +41,10 @@ actual constructor(
     }
 }
 
-@OptIn(ExperimentalMaterial3ExpressiveApi::class)
 internal actual fun createDefaultModalWideNavigationRailProperties() =
     ModalWideNavigationRailProperties()
 
 @OptIn(ExperimentalComposeUiApi::class)
-@ExperimentalMaterial3ExpressiveApi
```

Comment:
Why is it required? It's just about fixing a warning? Can we just ignore it and revert unrelated changes?

Reply:
Why did it work before?

Reply:
I'm aware of the changes in material3 and this annotation, I don't get why it break things only with updated compiler plugin?

Reply:
with old compiler plugin it is not possible to use this api either